6.0/Dockerfile

```diff
@@ -37,9 +37,11 @@ RUN set -x \
     && unzip sonarqube.zip \
     && mv sonarqube-$SONAR_VERSION sonarqube \
     && rm sonarqube.zip* \
-    && rm -rf $SONARQUBE_HOME/bin/*
+    && rm -rf $SONARQUBE_HOME/bin/* \
+    && mkdir -p $SONARQUBE_HOME/samples/conf \
+    && mv $SONARQUBE_HOME/conf/* $SONARQUBE_HOME/samples/conf
```

Comment:
@andreyhsiao I guess that this relocation here in order to pre-populate an empty volume, since volume hides existing content. I believed that in case of data-only containers good practice is to re-use image, so that data will be there. Thus best guess is that this is for the case of mount of a real file-system. Which I assume anyway not a good practice. So could you please elaborate a little bit more about this relocation?